### PR TITLE
Feature comparison tool sidebar

### LINKED ISSUE #185: Comparison Feature - Data Visualization
### Overview
Regarding #183 - UI is complete and up for review/next steps on development

### Action Items

- [ ] Requires front-end team to provide feedback

- [ ] Data team to confirm whether this shows what we want to show

- [ ] Project Manager to confirm this fits requirements

### Resources/Instructions
See Wireframe 3 page [here](https://www.figma.com/file/9RJJyaGAdT5hcRuv9PUMOY/311-Wireframe-v1.0?node-id=176%3A0)
Screenshot for frames below:
image
^Basic comparison form when "comparison button" is clicked
image
^ Options for frequency chart, time to close chart, and total request chart (not for how people contact 311 and not for type of request chart)

image
^Frequency Chart

image
^Time to close

image
^Total request

image
^How people contact 311

image
^Total Request

### PR DESCRIPTION
**Pull request by adamkendis**
Closes #185

Massive PR. All comparison tool filters, charts, charts placeholder, comparison backend routes and services, probably some other stuff lumped in.

### CODE REVIEW

**Comment by jmensch1** [comment]
hey @sellnat77, there's some new backend code in here...if you can find it

basically it's just three new endpoints for the comparison page, plus a new method in the DataService for the comparison filters

File: server/src/app.py [hunk 1]
```diff
@@ -75,21 +75,61 @@ def configure_app():
     return json(data)
 
 
+@app.route('/timetoclose-comparison', methods=["POST"])
```
Comment (sellnat77) [inline server/src/app.py:78]: Lets create a ticket to adjust our routes after presentation

im thinking something along the lines of `/comparison/timetoclose` that way we have `/timetoclose` and `/comparison/timetoclose`

**Comment by sellnat77** [comment]
@jmensch1 or @brodly backend stuff checks out
Frontend stuff is way over my head but when you approve it, feel free to merge it in

File: server/src/app.py (hunk 1 above)
Comment (jmensch1) [reply server/src/app.py:78]: sounds good, I can make it

**Pull request merged**